distro/archlinux/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: xatier
+_pkgname=fcitx5-mcbopomofo
+pkgname=fcitx5-mcbopomofo-git
+pkgver=97.f08da14
+pkgrel=1
+pkgdesc="McBopomofo for fcitx5"
+arch=('x86_64')
+url="https://github.com/openvanilla/fcitx5-mcbopomofo"
+license=('MIT')
+depends=('fcitx5')
+makedepends=('cmake' 'extra-cmake-modules')
```

Comment:
I know nothing about Arch, but on Ubuntu we need these packages in addition to `cmake` and `extra-cmake-modules`: `fcitx5 libfcitx5core-dev libfcitx5config-dev libfcitx5utils-dev gettext libfmt-dev`. Would any package maintainer (or user?) be able to successfully run the script by just having these two dependencies?

Reply:
Bacially `PKGBUILD` is a `bash` script that takes a few variables representing the metadata of the software [1]. `depends` should include the pakcages that `fcitx5-mcbopomofo` needs during build time and runtime.

Since `cmake` and `extra-cmake-modules` are not needed during the runtime, we can put them in `makedepends`. package managers (`pacman` and friends) would allow users to remove the `makedepends` after building the package.

I see the `.cmake` files are already installed by `fcitx5` package [2], btw.


[1] https://wiki.archlinux.org/title/PKGBUILD#Dependencies
[2] https://github.com/archlinux/svntogit-community/blob/packages/fcitx5/trunk/PKGBUILD

Reply:
Oh, forgot to mention that `gettext` is a dependency of `base` package, which should be installed on all Arch systems. `fmt` is a dependency of `fcitx5`.